crates/exex/types/src/finished_height.rs

```diff
@@ -23,3 +23,9 @@ impl FinishedExExHeight {
         matches!(self, Self::NotReady)
     }
 }
+
+impl From<u64> for FinishedExExHeight {
+    fn from(value: u64) -> Self {
+        Self::Height(value)
+    }
+}
```

Comment:
i'm not entirely convinced this is a good idea, we might add more variants that also just take a u64 so this is a potential future footgun. i get that this is a good idea right now, but i'd rather not have to break the api so "severely" in the future if possible

Reply:
right, good point. Can this PR be closed then?

Reply:
Feel free to close if this is a blocker, was just a small 5 min enhancement